scripts/DataStormUtil.py

```diff
@@ -52,7 +52,9 @@ class Reader(Server, DataStormProcess):
     processType = "reader"
 
     def __init__(self, instanceName=None, instance=None, *args, **kargs):
-        Server.__init__(self, *args, **kargs)
+        # Set readyCount to 0 to skip waiting for adapter activation, as a DataStorm reader may not activate any adapters
+        # when both DataStorm.Node.Server.Enabled and DataStorm.Node.Multicast.Enabled are set to 0.
+        Server.__init__(self, readyCount=0, *args, **kargs)
```

Comment:
This worked before because we always enabled the collocated adapter.

Reply:
I am not following:

> This worked before because we always enabled the collocated adapter.

You mean the tests or the code always enabled at least one adapter that was printing "Adapter ready"?

Since we no longer wait for any "Adapter ready" in the tests, there is no risk a test starts before DataStorm is actually ready?

Reply:
Before this change if you run a reader, with `DataStorm.Node.Server.Enabled` and `DataStorm.Node.Multicast.Enabled` both set to 0, the collocated adapter still printed ready.

But because I removed the call to `collocatedAdapter->ready` in DataStorm::Instance the reader would not print any ready messages in this case.

In the test suite Reader is defined as Server, and it always wait for readyCount adapters to be active, default to 1.

> Since we no longer wait for any "Adapter ready" in the tests, there is no risk a test starts before DataStorm is actually ready?

I have to check because one DataStorm test failed with the PR, but in most test we use the waitForReaders/waitForWriters which should be enough.